content/blog/2020/06/2020-06-14-repellent-terminology.adoc

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Repellent Terminology"
+tags:
+- community
+- opensource
+author: markyjackson-taulia
+description: "Terminology Updates"
+opengraph:
+  image: /images/docker/dockerJenkins_social.png
+---
+
+We have to talk, a real conversation about the power of words, honest talk about hurtful words, and the energy they hold.
+
+As a community, we cannot turn a blind eye to what is happening in the world. 
+What transpired on May 25th, 2020, and witnessed around the world has forever changed me as a person. 
```

Comment:
What transpired on May 25th, 2020, and has since been witnessed around the world, has forever changed me as a person.